description: no need to discuss the build process in binary file docs
author: Michael Gilbert <mgilbert@debian.org>

--- a/README
+++ b/README
@@ -18,36 +18,6 @@
 FAudio depends solely on SDL 2.0.9 or newer.
 FAudio never explicitly uses the C runtime.
 
-Building FAudio
----------------
-For *nix platforms, use cmake.
-
-    $ mkdir build/
-    $ cd build/
-    $ cmake ../
-    $ make
-
-For Windows, see the 'visualc/' directory.
-
-For Xbox One, see the 'visualc-winrt/' directory.
-
-For iOS/tvOS, see the 'Xcode-iOS/' directory.
-
-Unit tests
-----------
-FAudio includes a set of unit tests which document the behavior of XAudio2 and
-are to be run against FAudio to verify it has the same behavior. The tests are
-NOT built by default; set BUILD_TESTS=1 to build and then run the output with:
-
-    $ ./faudio_tests
-
-To build a Windows executable to run the tests against XAudio2, use the
-provided Makefile. This requires mingw-w64 to build.
-
-    $ cd tests/
-    $ make faudio_tests.exe
-    # run faudio_tests.exe on a Windows box
-
 Found an issue?
 ---------------
 Issues and patches can be reported via GitHub:
